core/src/com/unciv/ui/trade/OffersListScroll.kt

```diff
@@ -57,11 +64,19 @@ class OffersListScroll(val onOfferClicked: (TradeOffer) -> Unit) : ScrollPane(nu
                 val amountPerClick =
                         if (offer.type == Gold) 50
                         else 1
-                if (offer.amount > 0 && offer.name != Constants.peaceTreaty && offer.name != Constants.researchAgreement) // can't disable peace treaty!
+                if (offer.amount > 0 && offer.name != Constants.peaceTreaty && // can't disable peace treaty!
+                        offer.name != Constants.researchAgreement) {
+
+                    // highlight unique suggestions
+                    if (offerType in listOf(Luxury_Resource, Strategic_Resource)
+                            && otherOffers.all { it.type != offer.type || it.name != offer.name })
+                        tradeButton.color = Color.GREEN
```

Comment:
_actually_, I'd add `&& (offer.amount > 1 || offerType == Strategic_Resource )` to mark only your (or their) surplus, in other words, no immediate negative impact. From strategic resources, the maintenance is already deducted. From luxury, the ones providing happiness are not. Um, right?

Reply:
Good point

Reply:
Um, makes the sorting option mostly redundant, als long as ordering is predictable. Some wanted numbers on top to quickly see what can be given, others wanted side-by-side to quickly see the gaps in their luxuries - both ideas are covered by this. Genious. Simple but effective.
( oh, and I'd find `none { == && == }` much clearer )

Reply:
Sorry, I don't understand what you mean. What is a negative impact? Do you mean not trade the resources which are the only one? But probably, I may want to trade the single one for two others?

Reply:
When you trade away your last luxury resource, your happiness decreases. Impact. Not green, but you can still offer it. The effect of what you get in return isn't taken into account on that side of the table...